oak_loader/src/attestation.rs

```diff
@@ -50,7 +50,7 @@ pub async fn get_tls_identity_from_proxy(
         "Sending certificate signing request to Proxy Attestation Service: {:?}",
         uri
     );
-    let mut client = create_proxy_client(&uri, &root_tls_certificate)
+    let mut client = create_proxy_client(uri, root_tls_certificate)
```

Comment:
What was the error in this case? (if you remember :))

Reply:
The error was: "this expression borrows a reference (`&http::Uri` / `&[u8]`) that is immediately dereferenced by the compiler"